gramex/handlers/mlhandler.py

```diff
@@ -351,3 +351,16 @@ def delete(self, *path_args, **path_kwargs):
                 getattr(self, f'_delete_{item}')()
             except AttributeError:
                 raise HTTPError(BAD_REQUEST, f'Cannot delete {item}.')
+
+
+class MLPredictor(FormHandler):
+
+    @classmethod
+    def setup(cls, model, data, **kwargs):
+        cls.model = cache.open(model, joblib.load)
```

Comment:
Let's load the model on demand, AND via `gramex.cache.open` -- so if the file changes, we reload the model